examples/raycasting/main.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2016 The Ebiten Authors
```

Comment:
Happy new year, 2019 :-)

Reply:
I've never done a pull request before, not sure what happened when I clicked "resolve conversation". Do I just update locally and push again? :-)